kubenow/gluster-s3/templates/gluster-s3_bucket.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: batch/v1
```

Comment:
This file can be removed, it is better to give the user freedom to do this manually. The user may want to create N buckets and name them as they want. Please delete this, and see if you can create 2 or 3 buckets and access them with your favourite client from the master.

Reply:
Nice with the if template!